corehq/apps/app_manager/tests/test_get_questions.py

```diff
@@ -224,7 +224,8 @@ def test_get_questions_with_triggers(self):
         questions = form.wrapped_xform().get_questions(
             ['en', 'es'], include_triggers=True, include_translations=True)
 
-        self.assertEqual(questions, QUESTIONS)
+        for i in range(max(len(questions), len(QUESTIONS))):
```

Comment:
I'm curious, why did this change?

Reply:
@stephherbers I'm curious about this as well, was the original test not working?

Reply:
it wasn't working orginally, but now I just tested the original way and it's fine-- sorry about that switching it back